tools/scoring/score/parse_config.py

```diff
@@ -63,19 +65,30 @@ def __init__(self,
     }
     self.results = {}
 
-  # TODO: refactor into smaller functions and return instead of printing
-  def append_types(self):
+  @staticmethod
+  def append_types(
+      *, universe: ConfigUniverse,
```

Comment:
does `*` means any argument?

Reply:
No, it specifies that all arguments thereafter are keyword-only.